Packs/ThalesCipherTrustManager/Integrations/ThalesCipherTrustManager/ThalesCipherTrustManager.yml

```diff
@@ -0,0 +1,2389 @@
+      description: File extension
+      type: string
+    - contextPath: CipherTrust.LocalCA.id
+      description: A unique identifier for the certificate authority (CA).
+      type: String
+    - contextPath: CipherTrust.LocalCA.uri
+      description: Uniform Resource Identifier associated with the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.account
+      description: Account associated with the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.name
+      description: Name of the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.state
+      description: Current state of the CA (e.g., pending, active).
+      type: String
+    - contextPath: CipherTrust.LocalCA.createdAt
+      description: Timestamp of when the CA was created.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.updatedAt
+      description: Timestamp of last update of the CA.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.serialNumber
+      description: Serial number of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.subject
+      description: Subject of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.issuer
+      description: Issuer of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.notBefore
+      description: Start date of the CA's certificate validity.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.notAfter
+      description: End date of the CA's certificate validity.
+      type: Date
+    - contextPath: CipherTrust.LocalCA.sha1Fingerprint
+      description: SHA1 fingerprint of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.sha256Fingerprint
+      description: SHA256 fingerprint of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.sha512Fingerprint
+      description: SHA512 fingerprint of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.LocalCA.purpose.client_authentication
+      description: Indicates if client authentication is enabled for the CA.
+      type: String
+    - contextPath: CipherTrust.LocalCA.purpose.user_authentication
+      description: Indicates if user authentication is enabled for the CA.
+      type: String
+  - deprecated: false
+    description: Self-sign a local CA certificate. This is used to create a root CA. Either duration or notAfter date must be specified. If both notAfter and duration are given, then notAfter date takes precedence over duration. If duration is given without notBefore date, certificate is issued starting from server's current time for the specified duration.
+    name: ciphertrust-local-ca-self-sign
+    arguments:
+    - name: local_ca_id
+      isArray: false
+      description: An identifier of the resource. This can be either the ID (a UUIDv4),the Name, the URI, or the slug (which is the last component of the URI).
```

Comment:
```suggestion
      description: An identifier of the resource. This can be either the ID (a UUIDv4), the name, the URI, or the slug (which is the last component of the URI).
```